pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -61,6 +61,9 @@ type Kubernetes struct {
 // used when undeploying resources from kubernetes
 const TIMEOUT = 300
 
+//default value of Persistent Volume Claim
```

Comment:
this is not value of Persistent Volume Claim,  it should say that this is default size of Persistent Volume Claim